src/Stratis.Bitcoin.Features.PoA/PoABlockDefinition.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Features.MemoryPool;
+using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+using Stratis.Bitcoin.Features.Miner;
+using Stratis.Bitcoin.Mining;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.PoA
+{
+    public class PoABlockDefinition : BlockDefinition
+    {
+        /// <summary>Instance logger.</summary>
+        private readonly ILogger logger;
+
+        public PoABlockDefinition(
+            IConsensusManager consensusManager,
+            IDateTimeProvider dateTimeProvider,
+            ILoggerFactory loggerFactory,
+            ITxMempool mempool,
+            MempoolSchedulerLock mempoolLock,
+            Network network)
+            : base(consensusManager, dateTimeProvider, loggerFactory, mempool, mempoolLock, new MinerSettings(), network)
+        {
+            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+        }
+
+        /// <inheritdoc/>
+        public override void AddToBlock(TxMempoolEntry mempoolEntry)
+        {
+            this.AddTransactionToBlock(mempoolEntry.Transaction);
+            this.UpdateBlockStatistics(mempoolEntry);
+            this.UpdateTotalFees(mempoolEntry.Fee);
+        }
+
+        /// <inheritdoc/>
+        public override BlockTemplate Build(ChainedHeader chainTip, Script scriptPubKey = null)
+        {
+            this.OnBuild(chainTip, new Script());
+
+            this.coinbase.Outputs[0].ScriptPubKey = new Script();
+            this.coinbase.Outputs[0].Value = Money.Zero;
+
+            return this.BlockTemplate;
+        }
+
+        /// <inheritdoc/>
+        public override void UpdateHeaders()
+        {
+            base.UpdateBaseHeaders();
+
+            this.block.Header.Bits = Target.Difficulty1; // TODO POA use some constant difficulty all the time so chainwork rises
```

Comment:
Did you mean "Should use some constant difficulty so the chainwork **doesn't** rise"?
Good idea on the rule.

Reply:
Chainwork is a cumulative of difficulties of all blocks. So the chainwork HAVE to rise with every block. And it should rise by equal amount. 

For example we might have shorter chains on stratis that are better in terms of chainwork. For PoA logest chain will always mean the best chain.

Reply:
makes sense :-)